src/Tizen.Security.PrivacyPrivilegeManager/Tizen.Security/PrivacyPrivilegeManager.cs

```diff
@@ -180,6 +348,91 @@ public static WeakReference<ResponseContext> GetResponseContext(string privilege
+        ///     {
+        ///         for (int iterator = 0; iterator &lt; results.Lenght; ++itetartor)
+        ///         {
+        ///             switch(e.Results[iterator])
+        ///             {
+        ///
+        ///             case RequestResult.AllowForever:
+        ///                 Console.WriteLine("User allowed usage of privilege {0} definitely", e.Privileges[iterator]);
+        ///                 break;
+        ///             case RequestResult.DenyForever:
+        ///                  Console.WriteLine("User denied usage of privilege {0} definitely", e.Privileges[iterator]);
+        ///                 break;
+        ///             case RequestResult.DenyOnce:
+        ///                  Console.WriteLine("User denied usage of privilege {0} this time", e.Privileges[iterator]);
+        ///                 break;
+        ///             };
+        ///         }
+        ///      }
+        ///      else
+        ///      {
+        ///          Console.WriteLine("Error occured during requesting permission for {0}", string.Join(" ",e.Privileges));
+        ///      }
+        ///}
+        ///
+        ///     CheckResult[] results = PrivacyPrivilegeManager.CheckPermissions(privileges);
+        ///     List&lt;string&gt; privilegesWithAskStatus = new List&lt;string&gt;();
+        ///     for (int iterator = 0; iterator &lt; results.Lenght; ++itetartor)
+        ///     {
+        ///         switch (result)
+        ///         {
+        ///             case Allow:
+        ///                 // Privilege can be used
+        ///                 break;
+        ///             case Deny:
+        ///                 // Privilege can't be used
+        ///                 break;
+        ///             case Ask:
+        ///                 // User permission request required
+        ///                 privilegesWithAskStatus.Add(privileges[iterator]);
+        ///                 break;
+        ///         }
+        ///     }
+        ///
+        ///     PrivacyPrivilegeManager.ResponseContext context = null;
+        ///
+        ///     PrivacyPrivilegeManager.GetResponseContext(privilegesWithAskStatus.ToArray()).TryGetTarget(out context);
+        ///     if (context != null)
+        ///     {
+        ///         context.ResponseFetched += PPM_RequestMultipleResponses;
+        ///     }
+        ///
+        ///     PrivacyPrivilegeManager.RequestPermissions(privilegesWithAskStatus.ToArray());
+        ///
+        ///     PrivacyPrivilegeManager.GetResponseContext(privilegesWithAskStatus.ToArray()).TryGetTarget(out context);
+        ///     if (context != null)
+        ///     {
+        ///         context.ResponseFetched -= PPM_RequestMultipleResponses;
+        ///     }
+        /// </code>
+        /// </example>
+        /// <since_tizen> 5 </since_tizen>
+        public static WeakReference<MultipleResponseContext> GetResponseContext(string[] privileges)
```

Comment:
Why do you use WeakReference?

Reply:
1. To prevent memory leaks when handlers that are attached to event sources will not be destroyed at the same time.

Reference:
https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/aa970850(v=vs.100)
2. To be consistent with
```csharp
public static WeakReference<ResponseContext> GetResponseContext(string privilege)
```

Reply:
Why do you think so?
It is not meaory leaks, because it released by GC,
and why do you thinks it should be destroyed at the same time?

WeakReference should not used in API

Reply:
I know your intent but implemention was wrong

It should be like below changes
https://github.com/myroot/TizenFX/commit/9579961b3931dd52d92cb6e03f8f2a7f3b5a8cb1